### PR TITLE
Dịch "mf_vn" - Phần 2

### LINKED ISSUE #2676: Dịch "mf_vn" - Phần 2
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_recommender-systems/mf_vn.md#L105-L329

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by trGiang99**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Điền số issue sau dấu `#` (không có dấu cách): Close #2676

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy thảo luận thêm ở thẻ Discussions của dự án.

* [x] Format của bản gốc có được giữ nguyên không?
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn. Bạn đã kiểm tra rằng mỗi đoạn văn đã nằm một dòng?

* [x] Điền tên đầy đủ của bạn vào mục **Những người thực hiện** ở cuối tập tin.

## Lưu ý cho người phản biện (Reviewer)

1. Chỉ review khi Pull Request (PR) này đã sẵn sàng cho việc review. Một PR sẵn sàng khi PR đó không có label `status: wip`.
2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

### CODE REVIEW

File: chapter_recommender-systems/mf_vn.md [hunk 1]
```diff
@@ -175,15 +179,16 @@ class MF(nn.Block):
...
 
 
 <!--
 We then implement the RMSE (root-mean-square error) measure, which is commonly used to measure the differences between rating scores 
 predicted by the model and the actually observed ratings (ground truth) :cite:`Gunawardana.Shani.2015`. RMSE is defined as:
 -->
 
-*dịch đoạn phía trên*
+Tiếp theo, ta lập trình phép đo RMSE (*root-mean-square error* - căn bậc hai trung bình bình phương sai số), phép đo này được sử dụng rộng rãi nhằm đo sự khác nhau giữa giá trị đánh giá
+dự đoán và đánh giá được quan sát thực tế (nhãn chuẩn) :cite:`Gunawardana.Shani.2015`. RMSE được định nghĩa bằng:
```
Comment (cuongvng) [inline chapter_recommender-systems/mf_vn.md:191]:
```suggestion
Tiếp theo, ta lập trình phép đo RMSE (*root-mean-square error* - căn bậc hai trung bình bình phương sai số), phép đo này được sử dụng rộng rãi nhằm đo sự khác nhau giữa giá trị đánh giá dự đoán và đánh giá được quan sát thực tế (nhãn gốc) :cite:`Gunawardana.Shani.2015`. RMSE được định nghĩa bằng:
```

File: chapter_recommender-systems/mf_vn.md [hunk 2]
```diff
@@ -274,7 +281,8 @@ Finally, let us put all things together and train the model.
 Here, we set the latent factor dimension to 30.
 -->
 
-*dịch đoạn phía trên*
+Cuối cùng, hãy cùng gộp tất cả vào với nhau và huấn luyện mô hình.
```
Comment (cuongvng) [inline chapter_recommender-systems/mf_vn.md:284]:
```suggestion
Cuối cùng, hãy kết hợp tất cả với nhau và huấn luyện mô hình.
```

File: chapter_recommender-systems/mf_vn.md [hunk 3]
```diff
@@ -296,7 +304,7 @@ train_recsys_rating(net, train_iter, test_iter, loss, trainer, num_epochs,
 Below, we use the trained model to predict the rating that a user (ID 20) might give to an item (ID 30).
 -->
 
-*dịch đoạn phía trên*
+Ở dưới, ta sử dụng mô hình đã được huấn luyện để dự đoán đánh giá mà một người dùng (ID 20) có thể đánh giá một sản phẩm (ID 30).
```
Comment (cuongvng) [inline chapter_recommender-systems/mf_vn.md:307]:
```suggestion
Ở dưới, ta sử dụng mô hình đã được huấn luyện để dự đoán đánh giá có thể của một người dùng (ID 20) cho một sản phẩm (ID 30).
```

File: chapter_recommender-systems/mf_vn.md [hunk 4]
```diff
@@ -312,7 +320,8 @@ scores
 * We can implement and train matrix factorization for recommender systems.
 -->
 
-*dịch đoạn phía trên*
+* Mô hình phân rã ma trận được sử dụng rộng rãi trong hệ thống gợi ý. Nó có thể được sử dụng để dự đoán đánh giá mà một người dùng có thể đánh giá một sản phẩm.
```
Comment (cuongvng) [inline chapter_recommender-systems/mf_vn.md:323]:
```suggestion
* Mô hình phân rã ma trận được sử dụng rộng rãi trong hệ thống gợi ý. Nó có thể được sử dụng để dự đoán đánh giá có thể của một người dùng cho một sản phẩm.
```

File: chapter_recommender-systems/mf_vn.md [hunk 5]
```diff
@@ -323,7 +332,9 @@ scores
 * Check the predicted rating scores of other users for a specific movie.
 -->
 
-*dịch đoạn phía trên*
+* Thay đổi kích thước của nhân tố tiềm ẩn. Kích thước của nhân tố tiềm ẩn ảnh hướng thế nào đến hiệu năng của mô hình?
+* Thử các phép tối ưu, tốc độ học và tốc độ suy giảm trọng số khác nhau.
```
Comment (cuongvng) [inline chapter_recommender-systems/mf_vn.md:336]:
```suggestion
* Thử các bộ tối ưu, tốc độ học và tốc độ suy giảm trọng số khác nhau.
```

**Review by cuongvng — COMMENTED** [review]
Thanks!

File: chapter_recommender-systems/mf_vn.md [hunk 6]
```diff
@@ -138,7 +138,7 @@ npx.set_np()
 ## Model Implementation
 -->
 
-## *dịch tiêu đề trên*
+## Cách lập trình Mô hình
```
[the remark this answers is not in the document]
Comment (trGiang99) [reply chapter_recommender-systems/mf_vn.md:141]: Trong glossary, `Implementation` dịch là `Cách lập trình` mà anh.

File: chapter_recommender-systems/mf_vn.md (hunk 1 above)
Comment (trGiang99) [reply chapter_recommender-systems/mf_vn.md:191]: Câu này tách làm đôi á anh :D

File: chapter_recommender-systems/mf_vn.md [hunk 7]
```diff
@@ -348,6 +359,6 @@ Tên đầy đủ của các reviewer có thể được tìm thấy tại https
 * 
 
 <!-- Phần 2 -->
-* 
+* Đỗ Trường Giang
```
Comment (trGiang99) [inline chapter_recommender-systems/mf_vn.md:362]:
```suggestion
* Đỗ Trường Giang
* Nguyễn Văn Cường
```

File: chapter_recommender-systems/mf_vn.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (cuongvng) [reply chapter_recommender-systems/mf_vn.md:141]: chỗ này `cách` cũng được, mình chỉ rút ngắn đi thôi

File: chapter_recommender-systems/mf_vn.md (hunk 1 above)
Comment (cuongvng) [reply chapter_recommender-systems/mf_vn.md:191]:
```suggestion
Tiếp theo, ta lập trình phép đo RMSE (*root-mean-square error* - căn bậc hai trung bình bình phương sai số), phép đo này được sử dụng rộng rãi nhằm đo sự khác nhau giữa giá trị đánh giá
dự đoán và đánh giá thực tế (nhãn gốc) :cite:`Gunawardana.Shani.2015`. RMSE được định nghĩa bằng:
```

**Review by cuongvng — APPROVED** [review]

**Review by duythanhvn — APPROVED** [review]
